src/PrestaShopBundle/Form/Admin/Type/SearchProductType.php

```diff
@@ -36,7 +36,7 @@
 /**
  * Initiates input with ability to search for any type of product
  */
-class SearchProductType extends EntitySearchInputType
+class SearchProductType extends TranslatorAwareType
```

Comment:
symfony docs suggest not using inheritence, but add `getParent()` instead. That was part of the reason when I was getting duplicate block prefix errors